build-system/tasks/extension-generator/bento/amp-__component_name_hyphenated__/amp-__component_name_hyphenated__.md

```diff
@@ -139,6 +151,14 @@ Description of event. Use cases of event-name. Include all the nuances, such as:
 
 [/example]
 
+#### Bento mode
```

Comment:
Actions and events are not defined in Bento, hence I would remove it here. Instead describe the JS API in the standalone section. This keeps both sections simpler.

Reply:
Good call, this structure can easily get messy.